test/conformance/enqueue/urEnqueueUSMPrefetch.cpp

```diff
@@ -1,2 +1,82 @@
 // Copyright (C) 2023 Intel Corporation
 // SPDX-License-Identifier: MIT
+
+#include <uur/fixtures.h>
+
+using urEnqueueUSMPrefetchWithParamTest =
+    uur::urUSMDeviceAllocTestWithParam<ur_usm_migration_flag_t>;
+
+UUR_TEST_SUITE_P(urEnqueueUSMPrefetchWithParamTest,
+                 ::testing::Values(UR_USM_MIGRATION_FLAG_DEFAULT),
+                 uur::deviceTestWithParamPrinter<ur_usm_migration_flag_t>);
+
+TEST_P(urEnqueueUSMPrefetchWithParamTest, Success) {
+    ur_event_handle_t prefetch_event = nullptr;
+    ASSERT_SUCCESS(urEnqueueUSMPrefetch(queue, ptr, sizeof(int),
```

Comment:
could we make it so the `urUSMDeviceAllocTestWithParam` fixture has a variable denoting the size of the `ptr` allocation? instead of doing `sizeof(int)` everywhere

Reply:
Fair point. Maybe we can have an `allocation_size` or `ptr_size` member.

Reply:
I added an allocation_size variable to the fixture.